docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
 
----
+#### Compiling Android
+
+Android apps can have multiple different "variants" which are defined in the project's `build.gradle`. Variants can be selected with the `--variant` flag:
+
+<Terminal cmd={['$ npx expo run:android --variant debug']} />
+
+You can compile the Android app for production by running:
+
+<Terminal cmd={['$ npx expo run:android --variant release']} />
+
+This build is not guaranteed to be code signed for submission to the Google Play Store, this command should be used for testing bugs that may only show up in production builds. To generate a production build that is code signed for the Play Store, we recommend using `eas build -p android`.
+
+You can debug the project using native debugging tools by opening the `android/` folder in Android Studio:
+
+<Terminal cmd={['$ open -a /Applications/Android\ Studio.app android']} />
+
+### Exporting with Webpack
+
+> Webpack is only supported for the Web platform.
+
+You can export the JavaScript and assets for your web app using Webpack by running the following:
+
+<Terminal cmd={[
+'$ npx expo export:web',
+]} />
+
+- `--dev`: Bundle in 'development' mode without minifying code or stripping the `__DEV__` boolean.
+- `-c, --clear`: Clear the bundler cache before exporting.
+
+This command will be disabled if your project is configured to use `metro` for bundling web projects in the `app.json` via the `expo.web.bundler: 'metro'` field.
+
+## Prebuild
+
+<Terminal cmd={[
+'$ npx expo prebuild',
+]} cmdCopy="npx expo prebuild" />
+
+Before a native app can be compiled, the native source code must be generated. Expo CLI provides a unique and powerful system called _prebuild_, which generates the native code for your project.
+
+**Learn more**
+
+- [Prebuild](/guides/prebuild).
+- [Config plugins](/guides/config-plugins)
+- [Native modules API](/modules/module-api.md)
+
+## Telemetry
+
+Expo dev tools collect anonymous data about general usage, this helps us know when a command is safe to deprecate (low usage), or when a feature is not working as expected. Telemetry is completely optional and you can opt out by using the `EXPO_NO_TELEMETRY=1` environment variable.
+
+## Authentication
+
+Expo CLI provides authentication methods for use in the `npx expo start` command. Authentication is used to "code sign" manifests for secure OTA usage. Think of this like HTTPS in web.
+
+1. Register an account with `npx expo register`.
+2. Login to your account with `npx expo login`.
+3. Check which account is currently authenticated with `npx expo whoami`.
+4. Logout with `npx expo logout`.
+
+These credentials are shared across Expo CLI and EAS CLI.
+
+## Customizing
+
+Sometimes you may want to customize a project file which would otherwise be managed in-memory by the CLI. When you utilize tools other than Expo CLI, you'll need to have the default config files present, otherwise your app may not work as expected. You can generate files by running:
```

Comment:
```suggestion
Sometimes you may want to customize a project file that would otherwise be managed in-memory by the CLI. When you utilize tools other than Expo CLI, you'll need to have the default config files present, otherwise, your app may not work as expected. You can generate files by running:
```